jans-config-api/pom.xml

```diff
@@ -22,12 +22,33 @@
 		<surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
 
 		<jans.version>1.0.0-SNAPSHOT</jans.version>
-		<weld.version>3.1.2.Final</weld.version>
-		<jetty.version>11.0.2</jetty.version>
-		<resteasy.version>4.5.10.Final</resteasy.version>
+
+		<jetty.version>11.0.8</jetty.version>
+		<weld.version>4.0.3.Final</weld.version>
+		<resteasy.version>6.0.0.Final</resteasy.version>
+		
+		<jakarta.annotation.api.version>2.0.0</jakarta.annotation.api.version>
+		<jakarta.ejb.api.version>4.0.0</jakarta.ejb.api.version>
+		<jakarta.enterprise.cdi.api.version>3.0.0</jakarta.enterprise.cdi.api.version>		
+		<jakarta.inject.api.version>2.0.1</jakarta.inject.api.version>
+		<jakarta.validation.api.version>3.0.0</jakarta.validation.api.version>
+		<jakarta.ws.rs.api.version>3.0.0</jakarta.ws.rs.api.version>
+		<org.hibernate.validator.version>7.0.1.Final</org.hibernate.validator.version>
+		<org.hibernate.validator.cdi.version>7.0.1.Final</org.hibernate.validator.cdi.version>
+		<org.glassfish.jakarta.el.version>4.0.0</org.glassfish.jakarta.el.version>
+		<org.glassfish.jakarta.json.version>2.0.1</org.glassfish.jakarta.json.version>
+
 		<jackson.version>2.10.1</jackson.version>
-		<microprofile.version>3.0</microprofile.version>
-		<smallrye.version>2.2.0</smallrye.version>
+		<microprofile.config.version>3.0.1</microprofile.config.version>
+		<microprofile.health.version>4.0</microprofile.health.version>
+		<microprofile.metrics.version>4.0</microprofile.metrics.version>
+		<microprofile.rest.client.version>3.0</microprofile.rest.client.version>		
+		<smallrye.config.version>3.0.0-RC1</smallrye.config.version>
```

Comment:
We can import smallrye [bom ](https://mvnrepository.com/artifact/io.smallrye/smallrye-bom) file.